ubr/conf.py

```diff
@@ -120,23 +141,25 @@ def var(envname, cfgpath, default):
 # so we're explicit about which ones we're using.
 
 AWS = {
-    'aws_access_key_id': cfg('aws.access_key_id'),
-    'aws_secret_access_key': cfg('aws.secret_access_key'),
+    "aws_access_key_id": cfg("aws.access_key_id"),
+    "aws_secret_access_key": cfg("aws.secret_access_key"),
```

Comment:
any good reason to stick with one quote `'` or the other `"`, apart from consistency?

Reply:
I don't think or have an opinion any more. I defer individual thought regarding formatting in python to Black: https://github.com/psf/black

https://www.thoughtworks.com/radar/techniques/opinionated-and-automated-code-formatting